hack/olm-registry/olm-artifacts-template.yaml

```
@@ -85,6 +85,13 @@ objects:
         verbs:
         - get
         - create
+      - apiGroups:
+        - config.openshift.io
+        resources:
+        - proxies.config.openshift.io
```

```suggestion
        - proxies
```

Is .config.openshift.io needed here?